build/_build/Build.Steps.cs

```diff
@@ -254,6 +255,52 @@ partial class Build
             CopyFileToDirectory(source, dest, FileExistsPolicy.Overwrite);
         });
 
+    Target DownloadLibSqreen => _ => _
+        .Unlisted()
+        .After(CreateRequiredDirectories)
+        .OnlyWhenStatic(() => IsLinux)
+        .Executes(() =>
+        {
+            var wc = new WebClient();
+            var libSqreenUri = new Uri("https://www.nuget.org/api/v2/package/libsqreen/1.1.2.2");
```

Comment:
Given we have to manually download this for Linux, doesn't it make sense to remove it from the packages.config, and do the same thing on both platforms? That way we don't have to keep version numbers in sync etc?

Also, I'm thinking we should make this version number a const to be obvious?

Reply:
I've removed the predicate, so it'll run both under Windows and Linux, as I agree this will help build stability. I think we should keep the package.config reference as this will help people that want to use Visual Studio to build and run the tests.

Moved the version over to a constant too.